ethereum/core/src/main/java/tech/pegasys/teku/core/ForkChoiceUtil.java

```diff
@@ -434,6 +440,7 @@ private static AttestationProcessingResult indexAndValidateAttestation(
     return is_valid_indexed_attestation(targetState, indexedAttestation)
         .ifSuccessful(
             () -> {
+              attestation.setValidIndexedAttestation();
```

Comment:
I thought we needed to set this when we check the signatures in `AttestationValidator` and `AggregateAttestationValidator` from `AttestationManager`

Reply:
You're right I completely missed that.